auth-web/src/components/auth/create-account/GovmContactInfoForm.vue

```diff
@@ -0,0 +1,146 @@
+    <v-row >
+      <v-col cols="12" class="py-0 mb-4">
+        <h4
+          class="mb-1"
+        >Account Admin Contact</h4>
+      </v-col>
+    </v-row>
+    <!-- Email Address -->
+    <v-row>
+      <v-col cols="12" class="pt-0 pb-0">
+        <v-text-field
+          filled
+          label="Email Address"
+          req
+          persistent-hint
+          :rules="emailRules"
+          v-model="emailAddress"
+          data-test="email"
+          readonly
+          disabled
+        >
+        </v-text-field>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12" class="pt-0 pb-0">
+        <v-text-field
+          filled
+          label="Confirm Email Address"
+          req
+          persistent-hint
+          :error-messages="emailMustMatch()"
+          v-model="confirmedEmailAddress"
+          data-test="confirm-email"
+          readonly
+          disabled
+        >
+        </v-text-field>
+      </v-col>
+    </v-row>
+    <v-divider class="mt-7 mb-10"></v-divider>
+
+    <v-row>
+      <v-col cols="12" class="form__btns py-0 d-inline-flex">
+        <v-btn
+          large
+          depressed
+          color="default"
+          @click="goBack"
+          data-test="btn-back"
+        >
+          <v-icon left class="mr-2">mdi-arrow-left</v-icon>
+          <span>Back</span>
+        </v-btn>
+        <v-spacer></v-spacer>
+        <v-btn
+          large
+          color="primary"
+          class="save-continue-button mr-3"
+          @click="createAccount"
+          data-test="next-button"
+        >
+          <span>
+            Create Account
+            <!-- <v-icon class="ml-2">mdi-arrow-right</v-icon>s -->
+          </span>
+        </v-btn>
+        <ConfirmCancelButton
+          :showConfirmPopup="true"
+          :isEmit="true"
+          @click-confirm="cancel"
+        ></ConfirmCancelButton>
+      </v-col>
+    </v-row>
+
+  </v-form>
+</template>
+
+<script lang="ts">
+
+import { Component, Emit, Mixins, Prop, Vue } from 'vue-property-decorator'
+import CommonUtils from '@/util/common-util'
+import ConfirmCancelButton from '@/components/auth/common/ConfirmCancelButton.vue'
+import NextPageMixin from '@/components/auth/mixins/NextPageMixin.vue'
+import Steppable from '@/components/auth/common/stepper/Steppable.vue'
+import { User } from '@/models/user'
+
+import { namespace } from 'vuex-class'
+const userModule = namespace('user')
+
+@Component({
+  components: {
+    ConfirmCancelButton
+  }
+})
+export default class GovmContactInfoForm extends Mixins(NextPageMixin, Steppable) {
+  @userModule.Action('getUserProfile') public getUserProfile!: (identifer: string) => User
+
+  public emailAddress = ''
+  public confirmedEmailAddress = ''
+  // @Prop({ default: false }) isStepperView: boolean
+
+  $refs: {
+    form: HTMLFormElement
+  }
+
+  public emailRules = CommonUtils.emailRules()
+
+  public emailMustMatch (): string {
```

Comment:
If the fields are readonly why do we need rules? Also is there a better pattern to carve out the email and confirm email as a component and re-use everywhere. Feels like lot of duplicate codes.
Also in this case it's readonly, so no need for rules as this will only confuse devs.